### PR TITLE
Implement new design

### PR DESCRIPTION
**Pull request by PaulWagener**
Implements the design created by Bart Lenstra

See a working preview at: https://paulwagener.github.io/nl-covid19-notification-app-dashboard/

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -23,137 +24,168 @@ body {
...
-details.in-progress {
-    border-right-color: #ff9900;
+.statements details > summary::-webkit-details-marker {
+    display: none;
 }
 
-details.checked {
-    border-right-color: #00bb33;
+.statements summary:focus {
+    outline: none;
```
Comment (arianvp) [inline css/main.css:137]: Was al opgemerkt in de Slack; maar in kader van toegankelijkheid en ontdekbaarheid willen we waarschijnlijk wel wat meer stijl-hints dat se `<summary>` kan openklappen

Comment (PaulWagener) [reply css/main.css]: Excellent point, I’ve removed the offending CSS to improve accessibility.
A chevron or + symbol should probably also improve things, but I’ll leave that to someone who can do some actual graphic design for now.

**Comment by jsoeterbroek** [comment]
Should this (re-)design not better be reworked as a Jekyll '[theme](https://jekyllrb.com/docs/themes/)'? This will allow you later down the road to more easily swap out different look-and-feel options (during testing phase).

**Comment by bwbroersma** [comment]
@jsoeterbroek very fair point! In my change from plain HTML to Jekyll #3 I did not properly do this, sorry for that, but I thought the change was an improvement :slightly_smiling_face:. I think patches are welcome to split it to a theme!

**Comment by jsoeterbroek** [comment]
Never mind, custom jekyll theme won't work. Github supports only a subset of themes. my bad.

**Comment by bwbroersma** [comment]
@jsoeterbroek I think it is possible when I read the documentation:
> To use any other Jekyll theme hosted on GitHub, type `remote_theme: THEME-NAME`, replacing THEME-NAME with the name of the theme as shown in the README of the theme's repository. 

from [Adding a theme to your GitHub Pages site using Jekyll](https://help.github.com/en/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll#adding-a-theme)

**Comment by jsoeterbroek** [comment]
@bwbroersma Yes, you can use a remote theme, but this needs to be a public theme in a public gem repository, which is not what you want probably.

**Comment by bwbroersma** [comment]
@jsoeterbroek I don’t know the hassle of this, but some items (e.g. logo and fonts) can be kept out of the theme right? Then I don’t see the problem of people also reusing this theme etc. We can probably setup some GitHub Action to push updates to RubyGems. I’m new to this but it seems [pretty easy](https://webdesign.tutsplus.com/tutorials/how-to-create-and-publish-a-jekyll-theme-gem—cms-27475). Another option might be symlinks and/or submodules, but I don’t know how well this works on Window file systems.

**Comment by PaulWagener** [comment]
Refactoring the Jekyll templates into a theme is probably a great improvement. But I would suggest to keep PR’s small and focused and to keep this one just about the changes to the design. 
Theme refactoring can then be discussed in a separate issue or PR.

**Comment by bwbroersma** [comment]
Sorry to have this discussion in your PR @PaulWagener, luckily for you both @jsoeterbroek and I agree this with you :slightly_smiling_face:, I've a bit of a reminder of the endless `README.md` improvements in the HTML->Jekyll PR #3 :wink:

**Comment by jsoeterbroek** [comment]
Please merge: this PR is blocking for issues #16, #17, #18, #19, #20.

**Pull request merged**